packages/snaps-sdk/src/types/interface.ts

```diff
@@ -22,7 +27,12 @@ import { ComponentStruct } from '../ui';
  * either the value of an input or a sub-state of a form.
  */
 
-export const StateStruct = union([FileStruct, string(), boolean()]);
+export const StateStruct = union([
+  FileStruct,
+  string(),
+  boolean(),
+  CaipAccountIdStruct,
```

Comment:
Hmm, wondering if the additional type/validation is worth it here. `CaipAccountIdStruct` is a string at the end of the day anyway right?

Reply:
Oh I see @GuillaumeRx asked for this, care to elaborate?

Reply:
IMO this is better in terms of consistency and makes the types more clearer.

Reply:
Yeah, I understand, I'm just wondering if it introduces complexity for developers when trying to use the state values in their Snap 🤔